.github/workflows/bench.yml

```diff
@@ -0,0 +1,16 @@
+name: "Benchmarks"
+
+on: [pull_request]
+jobs:
+  lint:
+    name: Benchmarks
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Run Benchmark
+      run: |
+        docker run --rm -v $PWD:/app composer sh -c \
+        "composer install --profile --ignore-platform-reqs && git config --global --add safe.directory /app && composer bench -- --progress=plain"
```

Comment:
lets run this as a job in one of the existing workflows 
https://github.com/appwrite/appwrite/blob/main/.github/workflows/tests.yml#L102

Reply:
tests.yml runs for every service. This is perfectly fine to run only once. Separation also allows concurrency, I don't see downside in that. Plus it's copy&paste from utopia library, so it follows consistency to have it separated.

Reply:
I could see same discussion about linter - linter could be in tests.yml, but is separate.

Reply:
Removed bench CI/CD